torchgeo/datasets/openbuildings.py

```diff
@@ -231,9 +232,10 @@ def __init__(
            *root* was renamed to *paths*.
         """
         self.paths = paths
+        if isinstance(res, float):
+            res = (res, res)
```

Comment:
We also have a `torchgeo.samplers.utils._to_tuple` we could be using more. Maybe we should move that to `torchgeo.utils` or something to avoid circular imports. No action required here, just a thought about future code reuse.